pages/common/astyle.md

```diff
@@ -0,0 +1,24 @@
+# astyle
+
+> Source code indenter, formatter, and beautifier for the C, C++, C# and Java programming languages.
+> A copy of the original file is created with an dot orig appended to the original file name.
+
+- Apply default style, 4 spaces per indent and no formatting changes:
```

Comment:
Perhaps it would read better with `the default style of 4 spaces`.... instead of just `default style, 4 spaces`...?

Reply:
OK